packages/react-divider/src/components/Divider/Divider.types.ts

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
+import { ComponentProps } from '@fluentui/react-utilities';
+import { ColorTokens, RecursivePartial, FontTokens } from '@fluentui/react-theme-provider/lib/compat/index';
+
+/**
+ * {@docCategory Divider}
+ */
+export interface DividerProps extends ComponentProps, React.HTMLAttributes<HTMLElement> {
+  tokens?: RecursivePartial<DividerTokens>;
```

Comment:
```suggestion
```

`tokens` are not supported with current styling and can be removed for now.

Reply:
Updated